contentmgr/sanity_check.go

```diff
@@ -0,0 +1,28 @@
+package contentmgr
+
+import (
+	"github.com/application-research/estuary/util"
+	"github.com/ipfs/go-cid"
+)
+
+func (cm *ContentManager) HandleSanityCheck(cc cid.Cid, err error) {
+	// get all contents affected by this missing block on estuary or from shuttles
+	var cnts []util.Content
+	where := "id in (select content from obj_refs where object = (select id from objects where cid = ?))"
+	if err := cm.DB.Find(&cnts, where, cc.Bytes()).Error; err != nil {
+		cm.log.Errorf("sanity check failed to get content(s) for cid: %s, err: %w", cc.String(), err)
+		return
+	}
+
+	// mark all contents affected by this missing block
+	var marks map[uint]bool
+	for _, cnt := range cnts {
+		if _, ok := marks[cnt.ID]; !ok {
+			if err := cm.DB.Model(&util.Content{}).Where("id = ?", cnt.ID).Update("failed_sanity_check", true).Error; err != nil {
```

Comment:
Are we introducing a new field? I suggest you create a roll forward and back SQL script with transactions enabled.

Reply:
I'll add this to the instructions for the next release.

file_name: contents_add_failed_sanity_check_column_roll_forward.sql
```
\echo 'start - adding failed_sanity_check on contents table'
// Roll forward
alter table contents add failed_sanity_check boolean;
\echo 'end - adding failed_sanity_check on contents table'
```

file_name: contents_add_failed_sanity_check_column_roll_back.sql
```
\echo 'start - adding failed_sanity_check on contents table'
alter table contents drop column failed_sanity_check;
\echo 'end - adding failed_sanity_check on contents table'
```

Reply:
@alvin-reyes I think it best to use another table, so we can persist the missing block and the error message - which might come in handy for manually fixing the data, plus this won't strain the contents table